### PR TITLE
[Bugfix] Fix MR configuration problems

### PR DESCRIPTION
**Pull request by jerqi**
### What changes were proposed in this pull request?
We don't use RssConfig to override JobConf, Because some configuration will be overrided by default configuration. User configuration don't take effect.

### Why are the changes needed?
#163 introduced this problem. If we don't have this patch, some configuration will not take effect, like `io.sort.mb`.

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
Manual test

### CODE REVIEW

File: client-mr/src/main/java/org/apache/hadoop/mapreduce/RssMRUtils.java [hunk 1]
```diff
@@ -145,4 +145,24 @@ public static void applyDynamicClientConf(JobConf jobConf, Map<String, String> c
       }
     }
   }
+
+  public static int getInt(JobConf priorityConf, JobConf normalConf, String key, int defaultValue) {
```
Comment (frankliee) [inline client-mr/src/main/java/org/apache/hadoop/mapreduce/RssMRUtils.java:149]: The name of priorityConf and normalConf are strange.

Comment (jerqi) [reply client-mr/src/main/java/org/apache/hadoop/mapreduce/RssMRUtils.java:149]: `FirstConf SecondConf` may be better?

Comment (frankliee) [reply client-mr/src/main/java/org/apache/hadoop/mapreduce/RssMRUtils.java:149]: conf and defaultConf?

Comment (jerqi) [reply client-mr/src/main/java/org/apache/hadoop/mapreduce/RssMRUtils.java:149]: Not good. `rssJobConf and jobConf` ?

Comment (frankliee) [reply client-mr/src/main/java/org/apache/hadoop/mapreduce/RssMRUtils.java:149]: rssJobConf and mrJobConf ？

Comment (jerqi) [reply client-mr/src/main/java/org/apache/hadoop/mapreduce/RssMRUtils.java:149]: ok.

**Comment by frankliee** [comment]
LT2M

**Review by frankliee — APPROVED** [review]

**Pull request merged**